### PR TITLE
Improved doc according to US style

### LINKED ISSUE #12476: Improve Titles and Standardize Spelling According to US English
This issue aims to make minor improvements to the titles in the e.g. `advanced_topics/customisation` section of the documentation. While working on these  enhancements, the URL should also be updated to reflect US spelling ("customization" instead of "customisation") for consistency.

### PR DESCRIPTION
**Pull request by Srishti-j18**
Fixed #12476

- [X] Updated section titles from `Customisation` to` Customization`.
- [X] Replaced all occurrences of `customisation`, `customising`, `customised` etc, with `customization`, `customizing`, and `customized`, respectively, in the content.
- [X] Renamed URL paths from` /customisation/ `to` /customization/`.

### CODE REVIEW

**Comment by Srishti-j18** [comment]
Hi @lb- , Actually, I tried to fix for every occurrence, not just title ;) , and made changes to resolve conflict , but Idk why it's still showing here. Could you please check it?

**Comment by lb-** [comment]
@Srishti-j18 did you create a fresh branch from `main`?


You'll likely have to rebase on main and resolve issues, then force push to this branch.

File: docs/advanced_topics/third_party_tutorials.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ the latest Wagtail versions.
 -   [Deploy Wagtail CMS to PythonAnywhere using git push to a bare repo](https://www.nickmoreton.co.uk/articles/deploy-wagtail-cms-to-pythonanywhere/) (27 September 2023)
 -   [Building a custom Django command to check all admin pages are loading correctly in Wagtail](https://www.nickmoreton.co.uk/articles/wagtail-response-checker/) (1 September 2023)
 -   [Integrating ChatGPT with Wagtail for efficient content generation](https://octave.nz/post/integrating-chatgpt-with-wagtail-for-efficient-content-generation-and-content-management) (15 August 2023)
--   [Creating Wagtail StreamField StructBlocks with a Customised Editor Interface](https://enzedonline.com/en/tech-blog/how-to-create-wagtail-streamfield-structblocks-with-a-customised-editor-interface/) (9 July 2023)
+-   [Creating Wagtail StreamField StructBlocks with a Customized Editor Interface](https://enzedonline.com/en/tech-blog/how-to-create-wagtail-streamfield-structblocks-with-a-customized-editor-interface/) (9 July 2023)
```
Comment (lb-) [inline docs/advanced_topics/third_party_tutorials.md:27]: Please avoid updating external URLs. These are not hosted on our docs site, so this new link simply won't work.

**Comment by Srishti-j18** [comment]
Hi, I resolved the conflicts and also removed the update from external URLs.

File: docs/releases/6.3.md [hunk 2]
```diff
@@ -164,6 +164,11 @@ The [`WAGTAIL_AUTO_UPDATE_PREVIEW`](wagtail_auto_update_preview) setting has bee
 
 To disable the automatic preview update feature, set [`WAGTAIL_AUTO_UPDATE_PREVIEW_INTERVAL = 0`](wagtail_auto_update_preview_interval) in your Django settings instead.
 
+<<<<<<< HEAD
+=======
+## Upgrade considerations - changes affecting Wagtail customizations
+
+>>>>>>> Improved titles according to US style
```
[the remark this answers is not in the document]
Comment (Srishti-j18) [reply docs/releases/6.3.md]: Uhh, i guess it happened when rebased on the main branch and force-pushed to this branch , corrected now

File: docs/advanced_topics/third_party_tutorials.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ the latest Wagtail versions.
 -   [Deploying my Wagtail blog to Digital Ocean](https://rosederwelt.com/deploying-my-wagtail-blog-digital-ocean-pt-1/) Part 1 of a 2 part series (29 January 2020)
 -   [How to Create and Manage Menus of Wagtail application](https://www.accordbox.com/blog/wagtail-tutorial-12-how-create-and-manage-menus-wagtail-application/) - An updated overview of implementing menus (22 February 2020)
 -   [Adding a React component in Wagtail Admin](https://dev.to/lb/adding-a-react-component-in-wagtail-admin-3e) - Shows how to render an interactive timeline of published Pages (30 December 2019)
--   [Wagtail API - how to customise the detail URL](https://dev.to/wagtail/wagtail-api-how-to-customize-the-detail-url-2j3l) (19 December 2020)
+-   [Wagtail API - how to customise the detail URL](https://dev.to/wagtail/wagtail-api-how-to-customise-the-detail-url-2j3l) (19 December 2020)
```
Comment (lb-) [inline docs/advanced_topics/third_party_tutorials.md:141]:
```suggestion
-   [Wagtail API - how to customise the detail URL](https://dev.to/wagtail/wagtail-api-how-to-customize-the-detail-url-2j3l) (19 December 2020)
```

Sorry if my previous comments confused you, we should not be changing URLs for third party tutorials, these are external links and will not conform to our docs standards.

**Review by lb- — CHANGES_REQUESTED** [review]
Just one more fix up. The rest looks good. Thank you @Srishti-j18 

FYI @laymonage - this will trigger additional redirects required. What's the best way to track this kind of request?

File: docs/advanced_topics/third_party_tutorials.md (hunk 3 above)
Comment (Srishti-j18) [reply docs/advanced_topics/third_party_tutorials.md]: done

**Review by lb- — APPROVED** [review]
Thanks @Srishti-j18  - looking great now.

**Comment by laymonage** [comment]
> FYI @laymonage - this will trigger additional redirects required. What's the best way to track this kind of request?

@lb- I've re-invited you to our RTD project (the invite expired last time). After accepting the invite (please do so in the next 2 weeks), you can go to https://app.readthedocs.org/dashboard/wagtail/redirects/ to add the redirects. It's pretty barebones though, there are help docs linked at the bottom left of the sidebar but I'd be happy to answer any questions.

**Comment by lb-** [comment]
Thanks @laymonage  - I think I have done this now, please can you double check. https://app.readthedocs.org/dashboard/wagtail/redirects/24240/edit/

Example URLs that redirect (latest, not stable)

- https://docs.wagtail.org/en/latest/advanced_topics/customisation/
- https://docs.wagtail.org/en/latest/advanced_topics/customisation/custom_page_listings.html

I have also confirmed that other releases do not use the redirect.

**Comment by laymonage** [comment]
@lb- that looks good to me, thanks! Didn't know RTD supports `:splat` too 😄

**Pull request merged**